src/main.ts

```diff
@@ -1,18 +1,29 @@
 import * as core from '@actions/core'
 import got from 'got'
 
+function splitIds(ids: string): string[] {
+  return ids.split(',').map(id => id.trim())
+}
+
 async function run(): Promise<void> {
   try {
     const token = core.getInput('token', {required: true})
-    const participationIds = core
-      .getInput('participationId', {required: true})
-      .split(',')
-      .map(id => id.trim())
+
+    const participationIdInput = core.getInput('participationId')
+    const groupIdInput = core.getInput('groupId')
+    if (!participationIdInput && !groupIdInput) {
+      throw new Error(`participationId or groupId must be set`)
+    }
+    const participationIds = splitIds(participationIdInput)
+    const groupIds = splitIds(groupIdInput)
+
     const content = core.getInput('content', {required: true})
 
-    const requests = participationIds.map(participationId =>
+    const apiUrlBase = 'https://www.sonicgarden.world/room_api/v1'
```

Comment:
どちらかというと定数的な意味合いなので↓のほうが嬉しいです。

```suggestion
    const API_URL_BASE = 'https://www.sonicgarden.world/room_api/v1'
```